multiapps-controller-process/src/main/java/module-info.java

```diff
@@ -2,6 +2,7 @@
 
     exports org.cloudfoundry.multiapps.controller.process;
     exports org.cloudfoundry.multiapps.controller.process.client;
+    exports org.cloudfoundry.multiapps.controller.process.event;
```

Comment:
What do you think about naming the package `org.cloudfoundry.multiapps.controller.process.dynatrace`?
My point is that all the classes are related to the dynatrace configuration. Event sounds more like something that is part of the flowable.